mandatory/2-function-creation.js

```diff
@@ -15,15 +28,25 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate( num ){
+  if ( typeof num === "number" && num % 2 === 0 && num <= 100 ) {
```

Comment:
To keep things more readable it might be better to use predicates assigned to variables.

i.e

const isNumber = typeof num === "number";
const isEven = num % 2 === 0
const isLessThanOrEqualTo100 = num <= 100

if(isNumber && isEven && isLessThanOrEqualTo100)